library/private/camera_impl.h

```diff
@@ -35,28 +35,28 @@ class camera_impl : public camera
   /**
    * Documented public API
    */
-  camera& setPosition(const vector3_t& pos) override;
-  vector3_t getPosition() override;
-  void getPosition(vector3_t& pos) override;
-  camera& setFocalPoint(const vector3_t& foc) override;
-  vector3_t getFocalPoint() override;
-  void getFocalPoint(vector3_t& foc) override;
+  camera& setPosition(const point3_t& pos) override;
+  point3_t getPosition() override;
+  void getPosition(point3_t& pos) override;
+  camera& setFocalPoint(const point3_t& foc) override;
+  point3_t getFocalPoint() override;
+  void getFocalPoint(point3_t& foc) override;
   camera& setViewUp(const vector3_t& up) override;
   vector3_t getViewUp() override;
   void getViewUp(vector3_t& up) override;
-  camera& setViewAngle(const double& angle) override;
-  double getViewAngle() override;
-  void getViewAngle(double& angle) override;
+  camera& setViewAngle(const angleDeg_t& angle) override;
```

Comment:
I think it should be snake case: `angle_deg_t`. See https://en.cppreference.com/w/cpp/types/max_align_t for example

Reply:
why should typedef style be different than local variables ? Afaik, cppref seems to use snake case everywhere.

Reply:
If you choose to name the types with the _t suffix, it's to mimic the STL types, so we should use snake case in this case. I don't like point_3_t though, I think point3_t looks fine.